builder/builder.mk

```diff
@@ -268,10 +268,6 @@ ifneq ($(IS_PRIVATE),)
 else
 	@printf '$(CYN)==> $(GRN)recording $(BLU)%s$(GRN) => $(BLU)%s$(GRN) in S3...$(END)\n' "$$(git rev-parse HEAD)" $(patsubst v%,%,$(VERSION))
 	echo '$(patsubst v%,%,$(VERSION))' | aws s3 cp - 's3://$(AWS_S3_BUCKET)/dev-builds/'"$$(git rev-parse HEAD)"
```

Comment:
Based on your commit description, you are removing the push to S3 but isn't the copy to S3 done on this line or am I missing something?

Reply:
This is pushing an entry to the dev-build map in S3 – it's a way to look up the Docker tag pushed for a given GitHub SHA. This is distinct from pushing the chart, which the deletions below here are doing.

(We're going to get rid of the dev-build map, too, but that's a later PR.)

Reply:
ok, make sense 👍